demo/ContainerViewController.swift

```diff
@@ -353,3 +367,8 @@ extension UIViewController {
         viewController.removeFromParentViewController()
     }
 }
+
+private extension SafetyState.Object {
```

Comment:
What do you think about making color a computed property on `SafetyState.Object`? May be cleaner in usage code.